internal/pkg/cli/deploy/lbws.go

```diff
@@ -262,7 +265,12 @@ func (d *lbWebSvcDeployer) validateNLBRuntime() error {
 		logAppVersionOutdatedError(aws.StringValue(d.lbMft.Name))
 		return err
 	}
-	return validateLBWSAlias(d.lbMft.NLBConfig.Aliases, d.app, d.env.Name)
+	for _, listener := range d.lbMft.NLBConfig.NLBListeners() {
+		if err := validateLBWSAlias(listener.Aliases, d.app, d.env.Name); err != nil {
+			return fmt.Errorf(`validate 'nlb.alias': %w`, err)
```

Comment:
should this be something like:
```go
for i, listener := range d.lbMft.NLBConfig.NLBListeners() {
   err := fmt.Errorf(`validate "nlb.alias": %w`, err)
   if i > 0 {
       err = fmt.Errorf(`validate "nlb.additional_listeners[%d]": %w`, i-1, err)
   }
   return err
}
```

Reply:
Thinking about this some more, what does it to have a different alias per listener for NLB?

wouldn't all the alias names point to the same NLB endpoint 💭 I wonder if `alias` should remain a property only at the top level and not part of any listener :think